### DISCUSSION TITLE
[Feature] Memory as special album

### DISCUSSION THREAD

**Discussion by AlmightyFrog** [damongolding/immich-kiosk#242] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

How about a "special album" or mode to display the memory pictures as slide show?

---

I did write a small python script just the other day to fetch all the image urls to then show them randomly on a low powered device. I could totally get rid of it again and just rely on immich kiosk instead ding the "heavy" loading.
Then my esp32 just queries an image url like https://my-url/image?action=memory or whatever every 30 seconds and directly gets an jpeg.

**Comment by damongolding** [comment]
Are you referring to the memory-land api endpoint?

Happy to implement this, just need to decide if it would work as a special album or its own key e.g. `album=memories` or `memories=true`

**Comment by AlmightyFrog** [comment]
Thanks for quick answer. Yes i mean following API: https://immich.app/docs/api/get-memory-lane/
I guess best would be memories=true, as someone might create a album called "memories" maybe and that would then be unavaiable.

Would memories=true then also use e.g. the person filter? Could be interesting addition if it is optional.

**Comment by damongolding** [comment]
The album filter doesn't take in album names, only IDs and special keys words so that wouldn't be an issue but I do think `memories=true` is clearer then `album=memories`.

I'm not sure of the memory lane api offers a person filter 🤔

**Comment by AlmightyFrog** [comment]
You are right, there is as far as i know no inbuilt filter.
I did use in my script a query using https://immich.app/docs/api/get-asset-info for each picture checking and filtering manually the images.
But that of cause increases the load.

I guess not having a filter for that is totally fine, plain `memories=true` should be clean and sufficient.
